src/main/webapp/i18n/en/tutorialGroupImport.json

```diff
@@ -6,19 +6,19 @@
             "importButton": "Import Tutorial Groups",
             "dialogTitle": "Import Tutorial Group Registrations",
             "explanations": {
-                "description": "This dialog can be used to import tutorial groups and their registrations from a CSV file. Tutorial Groups will be created if they do not exist yet. Students will be deregistered from their current tutorial group and registered in the new one.",
-                "requirements": "Each row must always contain a value for the tutorial group title. If you want to not only create tutorial groups but also register students, the row must contain a value for either matriculation number or login. This is needed to identify the student. Values for last and first name are optional. The top row must specify the header!"
+                "description": "This dialog allows you to import tutorial groups and student registrations from a CSV file. If a tutorial group does not exist, it will be created. If students are listed in the csv then they will be moved from their current tutorial group to the new one specified in the CSV file.",
```

Comment:
```suggestion
                "description": "This dialog allows you to import tutorial groups and student registrations from a CSV file. If a tutorial group does not exist, it will be created. If students are listed in the CSV then they will be moved from their current tutorial group to the new one specified in the CSV file.",
```

Reply:
done 👍